tests/commands/arch.py

```diff
@@ -0,0 +1,47 @@
+"""
+Arch commands test module
+"""
+
+from tests.base import RemoteGefUnitTestGeneric
+from tests.utils import ARCH
+import pytest
```

Comment:
put third party libs in a separate paragraph before our libs.
stdlib
blank
pytest
blank
ours